QUARKUS-1325.md

```diff
@@ -0,0 +1,27 @@
+# QUARKUS-1325 - Helm chart support for Quarkus
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-1325
+
+This test plan cover Quarkus - Helm integration
+
+## Scope of the testing
+
+Verify a repeatable/versioned way to deploy Quarkus application into K8s/ocp through helm
+
+Test development will focus on
+- Generate and deploy a Quarkus Helm Chart over Openshift default repository
```

Comment:
Will you also cover deploying to external registry like quay.io?

Reply:
No, for this first release I was thinking to cover the default scenario which means: 
- Deploy app container image into docker.io | quay.io 
- install Helm charts into the (OpenShift) default helm chart repository

Reply:
Pls add `Deploy app container image into quay.io` detail

Reply:
This is still not fixed!